graphium/data/dataset.py

```diff
@@ -771,6 +476,30 @@ def __getitem__(self, idx):
 
         return datum
 
+def torch_enum_to_dtype(v: Union[int, torch.dtype]):
```

Comment:
It's used in collate.py, though I think I might have used it in dataset.py at some point.  I could move it to collate.py, if that would make more sense.  I tried to find if there's a function in PyTorch for doing this, instead of making my own, and there somewhat is, but only on the C++ side, not the Python side.